indico/modules/events/registration/forms.py

```diff
@@ -158,3 +158,23 @@ def validate_users_field(self, field):
         if existing:
             raise ValidationError(_("There are already registrations with the following email addresses: {emails}")
                                   .format(emails=', '.join(sorted(existing))))
+
+
+class EmailRegistrantsForm(IndicoForm):
+    from_address = SelectField(_("From"), [DataRequired()])
+    cc_addresses = EmailField(_("CC"),
+                              description=_("Beware, addresses in this field will receive one mail per registrant."))
+    subject = StringField(_("Subject"), [DataRequired()])
+    body = TextAreaField(_("Email body"), [DataRequired()], widget=CKEditorWidget(simple=True))
+
+    def __init__(self, *args, **kwargs):
+        super(EmailRegistrantsForm, self).__init__(*args, **kwargs)
+        from_addresses = ['{} <{}>'.format(session.user.full_name, email)
+                          for email in sorted(session.user.all_emails, key=lambda x: x != session.user.email)]
+        self.from_address.choices = zip(from_addresses, from_addresses)
+        self.body.description = render_placeholder_info('registration-email', registration=None)
+
+    def validate_email_body(self, field):
```

Comment:
wrong method name. should be `validate_body` (or you could rename `body` to `email_body` and `from_address` to `email_from` to keep the same names I already use for invitation emails - if you do that don't forget to update the donotsanitize thing)